### PR TITLE
feat(tags): `voice-debug` tag

### PR DESCRIPTION
**Pull request by JPBM135**
Discussed in discord
> https://canary.discord.com/channels/222078108977594368/1007612252486119426/1007612252486119426

### CODE REVIEW

File: tags/tags.toml [hunk 1]
```diff
@@ -1031,3 +1031,23 @@ const row = new ActionRowBuilder<SelectMenuBuilder>().addComponents(selectMenu)
...
+To debug your voice connection and player:
+- Use the `debug: true` when creating your `VoiceConnection` and `AudioPlayer`
+- Add an event listener to the <VoiceConnection> and the <AudioPlayer>: 
+```js
+// Add one for each class if applicable
+<AudioPlayer | VoiceConnection>
+    .on('debug', console.log)
+    .on('error', console.error)
+```
+- Add an `error` listener to the stream you are passing to the resource:
```
Comment (MrMythicalYT) [inline tags/tags.toml:1047]:
```suggestion
• Add an `error` listener to the stream you are passing to the resource:
```

File: tags/tags.toml [hunk 2]
```diff
@@ -1031,3 +1031,23 @@ const row = new ActionRowBuilder<SelectMenuBuilder>().addComponents(selectMenu)
 const row = new ActionRowBuilder<TextInputBuilder>().addComponents(textInput)
 ```
 """
+
+[voice-debug]
+keywords = ["voice-debug", "voice-debugging", "voice"]
+content = """
+To debug your voice connection and player:
+- Use the `debug: true` when creating your `VoiceConnection` and `AudioPlayer`
+- Add an event listener to the <VoiceConnection> and the <AudioPlayer>:
```
Comment (MrMythicalYT) [inline tags/tags.toml:1040]:
```suggestion
• Use `debug: true` when creating your `VoiceConnection` and `AudioPlayer`
• Add an event listener to the <VoiceConnection> and the <AudioPlayer>: 
```

File: tags/tags.toml [hunk 3]
```diff
@@ -1031,3 +1031,23 @@ const row = new ActionRowBuilder<SelectMenuBuilder>().addComponents(selectMenu)
...
+<AudioPlayer | VoiceConnection>
+    .on('debug', console.log)
+    .on('error', console.error)
+```
+- Add an `error` listener to the stream you are passing to the resource:
+```js
+<Stream>.on('error', console.error)
+```
+
+> Note: The `<>` represents classes that need to be adapted to their respective name on your code
```
Comment (MrMythicalYT) [inline tags/tags.toml:1052]:
```suggestion
> Note: The `<>` represents classes that need to be adapted to their respective name in your code
```

File: tags/tags.toml (hunk 2 above)
Comment (MrMythicalYT) [inline tags/tags.toml:1040]: Do the class names need the `<>` around them?

**Review by MrMythicalYT — CHANGES_REQUESTED** [review]

File: tags/tags.toml (hunk 2 above)
Comment (JPBM135) [reply tags/tags.toml:1040]: I personally find easier to understand

**Pull request merged**